### DISCUSSION TITLE
end-to-end annotations in BDD100k?

### DISCUSSION THREAD

**Discussion by neel04** [bdd100k/bdd100k#243] [General]
Hi, thanks for such a wonderfully comprehensive dataset! 🤗 

I had a quick question - Does BDD100K have steering/direct trajectory annotations? Apparently, according to the [DBNet](http://www.dbehavior.net/data/egpaper_release.pdf) paper there is indeed; However, the documentation only mentions GPS trajectories and IMU data 🤔

So is there any such data to train models end-to-end with this dataset (for *imitation learning*), like the Comma.ai dataset (which has direct steering angles)?

**EDIT:** This is specifically regarding the annotations for the larger set of 100k video clips

**Comment by huydung179** [comment]
Hello,
Have you found the answer for this question?
